node/src/accountant/mod.rs

```diff
@@ -255,8 +361,11 @@ impl Accountant {
             earning_wallet: config.earning_wallet.clone(),
             payable_dao: payable_dao_factory.make(),
             receivable_dao: receivable_dao_factory.make(),
+            pending_payments_dao: pending_payments_dao_factory.make(),
             banned_dao: banned_dao_factory.make(),
             crashable: config.crash_point == CrashPoint::Message,
+            scanners: Scanners::default(),
+            transaction_confirmation: TransactionConfirmationTools::default(),
```

Comment:
I think I'd like to see this called "transaction_confirmation_tools," or even just "tools."